modules/virt-virtctl-commands.adoc

```diff
@@ -174,6 +174,24 @@ Optional:
 
 |`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file> --volume=<volume_name>`
 |Create a `VirtualMachineExport` CR and then download the volume defined in the CR.
+
+|`virtctl vmexport download export --manifest`
+|Retrieve the manifest for an existing export. The manifest does not include the header secret.
```

Comment:
Do we want to explain what a 'header secret' is? It is a token that allows access to the endpoint in a format that can used by OCP-V to import the disk image.

Reply:
What about the `service-url` flag? Maybe we could add some examples for it too.

Reply:
I guess we could, but in OpenShift that should not be needed since it is automatically detected from the Route

Reply:
@awels 
Am I understanding correctly: 
Yes to adding a description of header secret?
No to adding service-flag info?

Reply:
Yes on adding the description of the header secret since that is not a standard thing, so people won't be familiar with it. I am on the fence on the service-url flag, in general people shouldn't need to use it because it is auto detected, but I can imagine scenarios where it would be needed, so I am not entirely sure.